Automattic/jetpack
# Avoid using getBlockFromExample, unsupported on WP 5.2

#### Changes proposed in this Pull Request:


* Stop using `getBlockFromExample` which is not supported on WordPress 5.2.

In https://github.com/Automattic/jetpack/pull/14075/commits/1594818649d4431e8e769c1bed74adaec0f1a2f2 we changed the embed type preview to be rendered from the block example attributes instead of using a simple image.
As pointed out in https://github.com/Automattic/jetpack/issues/14466, `getBlockFromExample` has been introduced after WP 5.2, so the block would break with that WP version.

I've basically reverted that change and reintroduced the old image, but more optimized (2KB instead of 16), and with a bit of CSS improvements.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Bug fix

#### Testing instructions:






* On a Jetpack Docker environment, run `yarn docker:wp core update --version=5.2 --force` to downgrade to WP 5.2.
* Open the editor, insert an Eventbrite block, and add an Eventbrite event URL.
* Check in the block sidebar that the "Embed type" section looks ok at all screen sizes.
* Save and reload: make sure the block show up and works as expected.

#### Proposed changelog entry for your changes:

* N/A
